charts/ingress-nginx/values.yaml

```diff
@@ -362,6 +362,7 @@ controller:
         repository: jettech/kube-webhook-certgen
         tag: v1.2.0
         pullPolicy: IfNotPresent
+        runAsUser: 2000
```

Comment:
I think that this should not be under `image:` Lets move it out of here and into `patch:` directly.

Reply:
Fixed.